lighthouse-extension/gulpfile.js

```diff
@@ -135,10 +146,15 @@ gulp.task('browserify-lighthouse', () => {
       bundle.ignore(require.resolve('../lighthouse-core/gather/connections/cri.js'));
 
       // Prevent the DevTools background script from getting the stringified HTML.
-      if (/lighthouse-background/.test(file.path)) {
+      if (isDevtools(file.path)) {
```

Comment:
just verified it would still get this since report-generator is used by Runner.

sg!